pom.xml

```diff
@@ -24,6 +24,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${project.build.sourceEncoding}</project.reporting.outputEncoding>
         <java.version>1.7</java.version>
+        <postgresql.driver.version>9.4.1211</postgresql.driver.version>
```

Comment:
I'm surprised by seeing these changes here. They obviously come by merging the master in to you PR. I would prefer to use rebase in such situations to avoid to mix changes from other PRs into you PR. That is not a complained against this PR, it is more a general note.